### DISCUSSION TITLE
[Feature Request] Allow using Pyright from its MAIN branch

### DISCUSSION THREAD

**Discussion by Azureblade3808** [erictraut/pyright-playground#3] [Ideas]
Sometimes I find an issue and suspect that it might be a duplicate of an existing issue and have already been fixed. It would be convenient to verify if the MAIN branch version could be used in the playground.

**Comment by erictraut** [comment]
This feature would involve building the project from scratch on the pyright playground server. This would lead to very long delays and would be computationally expensive. The current implementation of pyright playground is designed to leverage caching so it is computationally efficient and generally fast. Since I'm personally paying the monthly cloud bill for the pyright playground, I'm not interested in increasing the compute costs, which have been rising monthly, so I'm not inclined to implement such a feature.

**Comment by Azureblade3808** [comment]
Understood. I'll try other means.
